vulkan/vulkan_utilities.cc

```diff
@@ -43,13 +43,9 @@ static std::vector<std::string> InstanceOrDeviceLayersToEnable(
   // NOTE: The loader is sensitive to the ordering here. Please do not rearrange
   // this list.
 #if OS_FUCHSIA
-  // Fuchsia uses the updated Vulkan loader and validation layers which no
-  // longer includes the image validation layer.
+  // Use the unified VK_LAYER_KHRONOS_validation
   const std::vector<std::string> candidates = {
```

Comment:
can you add a comment here indicating that we should not include other `candidates` for future reference.